### ISSUE TITLE
Feat: Make some of the new features deactivable

### ISSUE THREAD

**Issue by inkhey** [tracim/tracim#4388]
## Feature description and goals

New features like:
- link preview
- favorites

are not deactivable for now. We should consider making them deactivable.
Some discussion need to be done to find the proper way to do so.

**Comment by lebouquetin** [comment]
First discussion need to be done about why to allow to deactivate these features, not "how"

**Comment by inkhey** [comment]
> First discussion need to be done about why to allow to deactivate these features, not "how"

Yes and which feature make sense to be deactivable, as we may have different decision for different features.

**Comment by inkhey** [comment]
> > why to allow to deactivate these features
> 
> This is probably what @inkhey meant by "We should consider making them deactivable" (I don't read it as "we must do it")
> 
> As for reasons why, people might not want their instance to make requests to third party servers, or loading images from third parties client-side (but this can already be blocked using CSPs). I don't see a cons for this, except for dev time (which can be purely backend, the current code from the frontend will do requests for link previews to the backend and no preview will be shown if the backend answers empty preview data).

I see a cons, in the fact that this modularity is a point which may be hard to test properly, and we may expect weird bug is some specific configurations.
